src/ApiCompat/Microsoft.DotNet.ApiCompatibility/Microsoft.DotNet.ApiCompatibility.csproj

```diff
@@ -10,6 +10,10 @@
     <NoWarn>$(NoWarn);RS1024</NoWarn>
   </PropertyGroup>
 
+  <ItemGroup Condition="'$(TargetFrameworkIdentifier)' == '.NETFramework'">
+    <Compile Include="$(RepoRoot)src\Common\NullableAttributes.cs" LinkBase="Common" />
```

Comment:
For my own education, what's this file's purpose?

Reply:
Nullable attributes used for using nullable features on older frameworks.